beluga/test/beluga/sensor/data/test_laser_scan.cpp

```diff
@@ -34,9 +34,7 @@ class SimpleLaserScan : public beluga::BaseLaserScan<SimpleLaserScan> {
   using Scalar = double;
 
   SimpleLaserScan(std::vector<double> ranges, std::vector<double> angles, Sophus::SE3d origin = Sophus::SE3d{})
-      : ranges_{std::move(ranges)}, angles_{std::move(angles)}, origin_{std::move(origin)} {
-    assert(scan_ != nullptr);
```

Comment:
@nahueespinosa uh, how did CI pass? Oh! Are we building in release only? We should probably do both release and debug, or debug only (to not preprocess away all asserts).

Reply:
+1